authors/apps/articles/views/comments.py

```diff
@@ -260,34 +172,40 @@ def put(self, request, *args, **kwargs):
         # the comment, and has the authority to update the comment. If this check
         # fails, we inform the user that this action is forbidden.
         if comment.user.id != request.user.id:
-            return Response(
-                {
+            error_response = {
+                "response": {
                     "user": {
                         "user": RESPONSE['not_found'].format(data="User")
```

Comment:
I believe that this should be `unauthorized` and not a `not_found`

Reply:
@mutoromike  we have updated it to use forbidden for a message we have in response.py